CONTRIBUTING.md

```diff
@@ -135,4 +135,10 @@ To make this easier, we do appreciate it if folks enable the "Allow edits from m
 
 ## On ideologically motivated changes
 
-This is a purely technical project. As such, it is not an appropriate arena to advertise your personal politics or religious beliefs. Any changes that appear ideologically motivated will be rejected.
+Serenity's goal is to enable collaboration between as many groups as *reasonably* possible, and we welcome contributions that make the project more accessisble to people.
+
+However, Serenity is intended to be a purely technical exercise, in the sense of it not being meant to invoke any sociopolitical change. We explicitly try to avoid drama or getting involved in "external" culture wars, and can reject changes that we feel to be sensitive.
+
+For instance, we can reject changes promoting dogwhistles, religious beliefs—which are unambiguously off-topic—or real-world political candidates.
+
+We may miss the mark sometimes, but we encourage good-faith dialogue over anger.
```

Comment:
i feel like this sentence can be grossly misinterpreted if bad actors were to actually start contributing into the project. (i.e. "it's okay to be bad sometimes because everyone does that AND we have that in the rules!")

I'd reword this as `We're hoping to create a space where everyone is welcome to contribute. In a big project, that's not an easy task - we highly encourage good-faith dialogue over anger and assumption of guilt.` or something along those lines.

Reply:
you're right, this leaves space for tone-policing

Reply:
'Tone policing' is a good thing. The most important time to be calm and measured is when you are angry about something. These extra sentences should just not be added. They contribute nothing.

Reply:
"Tone policing" is a bad thing, it encurages off-topic discussions while usually being vague enough to not being able to outright invoke a proper code of conduct discussion (or discarding inappropriate comments) or (default to) this minimal ethics primer discussed here.

Reply:
I think it's very cheeky and dismissive of you to assume that you need to give anyone else a 'basic ethics primer'. That kind of attitude is just what people are trying to avoid when they say 'keep politics out of this'. As soon as some people get political they lose all perspective, all respect for others, and all sensible ideas about how other people can be expected to act. 

You don't need a complex rulebook or a big list of rights or rules or an 'ethics primer' to kick out people that are being obnoxious. You just need someone to do it. Anyone that cries 'but the rules are too vague!' instead of 'gosh sorry I really didn't mean to be rude' isn't worth keeping around.